examples/with-redux-toolkit/lib/slices/clockSlice.js

```diff
@@ -0,0 +1,31 @@
+import { createAction, createReducer } from '@reduxjs/toolkit';
+
+/**
+ * @param {boolean} light
+ * @returns {Object}
+ */
+export const tick = createAction('clock/tick', function prepare(light) {
```

Comment:
I'd still prefer to see `createSlice` used consistently.  If you want to stick with `createReducer/Action` here, can you add a comment explaining why you're doing it differently in the two files?